rspec-core/lib/rspec/core/configuration.rb

```diff
@@ -1773,6 +1773,34 @@ def warnings?
         $VERBOSE
       end
 
+      # Set Ruby deprecation warnings on or off.
+      def deprecation_warnings=(value)
+        @deprecation_warnings_set = true
+        if defined?(::Warning) && ::Warning.respond_to?(:[]=)
+          ::Warning[:deprecated] = value
+        end
+      end
+
+      # @return [Boolean] Whether or not ruby warnings are enabled.
+      def deprecation_warnings?
+        if defined?(::Warning) && ::Warning.respond_to?(:[]=)
+          ::Warning[:deprecated]
+        else
+          false
+        end
+      end
+
+      # @private
+      def deprecation_warnings_set?
```

Comment:
Would [:nocov:](https://github.com/simplecov-ruby/simplecov?tab=readme-ov-file#ignoringskipping-code) help here?

Reply:
Ah TIL, thank you!